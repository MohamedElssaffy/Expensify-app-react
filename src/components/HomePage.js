import React from 'react';
import ExpenseFilters from './ExpenseFilters';
import ExpenseList from './ExpenseList';

const HomePage = () => (
  <div>
    <ExpenseFilters />
    <ExpenseList />
  </div>
);

export default HomePage;
