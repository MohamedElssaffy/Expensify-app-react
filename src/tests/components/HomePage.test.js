import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../../components/HomePage';

test('Should render HomePage', () => {
  const wrapper = shallow(<HomePage />);

  expect(wrapper).toMatchSnapshot();
});
