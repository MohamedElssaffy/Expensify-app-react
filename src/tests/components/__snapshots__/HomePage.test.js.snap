// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render HomePage 1`] = `
<div>
  <Connect(ExpenseFilters) />
  <Connect(ExpenseList) />
</div>
`;
